pkg/extract/types/data/detection/criteria/criterion/criterion.go

```diff
@@ -16,10 +16,24 @@ import (
 
 type Criterion struct {
 	Vulnerable bool                    `json:"vulnerable,omitempty"`
+	FixStatus  FixStatus               `json:"fix_status,omitempty"`
```

Comment:
If vulnerable = false, FixStatus will not be set, right?
```suggestion
	FixStatus  *FixStatus               `json:"fix_status,omitempty"`
```

Reply:
Good catch!